recipes/py3nvml/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "py3nvml" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/fbcotter/py3nvml/archive/{{ version }}.tar.gz"
+  sha256: "6f111b8bc460228ca5c5b430f80d0dbc27b11e8ad5a589bd0be51fae879bb75e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - enum34
+    - xmltodict
```

Comment:
Would drop anything from here that is not needed for the build (like `enum34` and `xmltodict`).

Reply:
Removed everything except pip and python as suggested